CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ endif()
 add_sanitize_options()
 add_sanitize_blacklist(config/sanitizers/ubsan_blacklist.txt.in)
 
+# Without these, build tree shared libraries are not used on a machine where Katana is already installed
+SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--disable-new-dtags" PARENT_SCOPE)
```

Comment:
should this put in `BuildCommon.cmake` instead?

Reply:
Sure, moved.